src/index.ts

```diff
@@ -5,9 +5,4 @@ export { useQueryParam } from './useQueryParam';
 export { useQueryParams } from './useQueryParams';
 export { withQueryParams, InjectedQueryProps } from './withQueryParams';
 export { QueryParams, QueryParamsProps, QueryRenderProps } from './QueryParams';
-export { updateUrlQuery } from './updateUrlQuery';
-export {
-  QueryParamProvider,
-  QueryParamContext,
-  QueryParamContextValue,
-} from './QueryParamProvider';
```

Comment:
The changes I made modify the parameters of `updateUrlQuery` and the signature of `QueryParamContext` and the type `QueryParamContextValue`, which are library exports. However, these aren't documented in the README or included in any examples. Would taking these out of the library exports be considered a breaking change? They seem like internal details of the library to me.

Reply:
I think that's fine